app/models.py

```diff
@@ -9,7 +9,7 @@
 from notifications_utils.letter_timings import get_letter_timings
 from notifications_utils.recipient_validation.email_address import validate_email_address
 from notifications_utils.recipient_validation.errors import InvalidRecipientError
-from notifications_utils.recipient_validation.phone_number import PhoneNumber
+from notifications_utils.recipient_validation.notifynl.phone_number import PhoneNumber
```

Comment:
is it an idea to comment out non-nl references rather than delete them so we at least see that we replaced something here.. not sure.. lets discuss